test/integration/images/sti-fake-onbuild/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM sti_test/sti-fake
+
+ONBUILD RUN touch /sti-fake/src/onbuild
+
+RUN touch /sti-fake/src/index.html
+
+ADD scripts/.sti/bin/ /tmp/src/	
```

Comment:
Almost forgot, make sure above `ADD` command corresponds to `io.openshift.s2i.scripts-url` label.